CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ include(${CMAKE_SOURCE_DIR}/4build/cmake/CUDA.cmake)
 
 find_package(PythonLibs 2.7 REQUIRED)
 find_package(Numpy REQUIRED)
+find_package(OpenBLAS REQUIRED)
```

Comment:
http://cmake.org/cmake/help/v3.0/command/find_package.html: "The REQUIRED option stops processing with an error message if the package cannot be found." Guess you should remove this here, otherwise it will never reach `if(NOT OpenBLAS_FOUND)`.